src/components/Slider.js

```diff
@@ -18,6 +22,8 @@ class Slider extends React.Component {
     let { value } = e.target;
     this.props.setTargetVolume(value / 10);
     this.setState({ sliderVal: e.target.value });
+    // Save volume preference
+    store.set('coderadio-volume', value);
```

Comment:
you should set the volume preference in the main component in setTargetVolume, so when users change their volume using shortcuts, they would be able to save to storage.